test/autoCompletion.test.ts

```diff
@@ -1840,6 +1840,33 @@ describe('Auto Completion Tests', () => {
         createExpectedCompletion('kind', 'kind', 0, 0, 0, 2, 10, InsertTextFormat.Snippet, { documentation: '' })
       );
     });
+
+    it('should use markdownDescription for property completion', async () => {
+      languageService.addSchema(SCHEMA_ID, {
+        type: 'object',
+        properties: {
+          kind: {
+            type: 'string',
+            description: 'Kind is a string value representing the REST',
+            markdownDescription:
+              '**kind** (string)\n\nKind is a string value representing the REST resource this object represents.',
+          },
+        },
+        required: ['kind'],
+      });
+
+      const content = 'kin';
```

Comment:
Should `kin` be `kind`?

Reply:
It should be `kin` to test completion to `kind`

Reply:
ok